types/pigpio-dht/index.d.ts

```diff
@@ -0,0 +1,39 @@
+// Type definitions for pigpio-dht 1.1
+// Project: https://github.com/depuits/pigpio-dht
+// Definitions by: Erik Mavrinac <https://github.com/erikma>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+export = dht;
+
+/**
+ * Returns a new DHT object for accessing a DHT11 or DHT22/AM2302 via GPIO.
+ * @param gpio      an unsigned integer specifying the GPIO number
+ * @param type      11 to initialize for reading a DHT11, 22 for DHT22/AM2302
+ */
+declare function dht(gpio: number, type: number): dht.Dht;
```

Comment:
You could restrict the type to `type: 11 | 22`, although this will require a cast from users who aren't passing consts or literals. (Something like `dht(n, m as 11 | 22)`)

Reply:
Neat, didn't know you could do that, but I think I prefer to keep away casts in case someone is using a variable to feed in, e.g. a config scenario that selects the matching number.